doc/api/fs.md

```diff
@@ -302,6 +302,44 @@ Reads data from the file and stores that in the given buffer.
 If the file is not modified concurrently, the end-of-file is reached when the
 number of bytes read is zero.
 
+#### `filehandle.readableStream()`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* Returns: {ReadableStream}
+
+Returns a `ReadableStream` that may be used to read the files data.
+
+This method can only be called once, and only if the `FileHandle`
+is not closed or closing.
```

Comment:
What happens if this method is called more than once?

Reply:
An error is throw. I'm clarifying in the docs.